cookbook/core/control_flow/subworkflows.py

```diff
@@ -100,6 +110,86 @@ def nested_parent_wf(a: int) -> Tuple[int, str, str, str]:
 
 
 # %%
+# You can execute the nested workflows locally as well.
 # You can execute the nested workflows locally as well
 if __name__ == "__main__":
     print(f"Running nested_parent_wf(a=3) {nested_parent_wf(a=3)}")
+
+
+# %%
+# Child Workflow
+# ^^^^^^^^^^^^^^
+# When using LaunchPlans within a workflow to launch an execution of a previously defined workflow, a new
+# external execution is launched, with a separate execution ID and can be observed as a distinct entity in
+# FlyteConsole/Flytectl etc. Moreover, the context is not shared, which means they may have separate parallelism constraints.
+# We refer to these externalized invocations of a workflow using launch plans from a parent workflow as ``Child Workflows``.
```

Comment:
Isn't this paragraph the same as the one in the "Why should I use subworkflows" section? Should we be removing it there?